### PR TITLE
Update repo to match demo app

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-2850](https://team-1624093970686.atlassian.net/browse/AR-2850).

## Changes

- Copy all the code from https://github.com/arcana-network/demo-app
- Remove SDK integrations (arcana/auth and arcana/wallet) and add comment based instructions

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,6 +18,9 @@
 
   <body>
     <div id="app"></div>
+    <!-- SDK-1: Include Bundled SDK scripts -->
```
Comment (karthik-durai) [inline index.html]: remove this comment

**Review by karthik-durai — APPROVED** [review]

File: src/services/auth.service.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    // ...
+  }
+
+  async function isLoggedIn() {
+    // AUTH-3: Check if the user is already logged in.
+    // ...
+  }
+
+  async function requestSocialLogin(type) {
+    // AUTH-4: Sign in user with SSO with selected login type.
```
Comment (karthik-durai) [inline src/services/auth.service.js]: replace SSO with OAuth

File: src/services/auth.service.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+// AUTH-IMPORT: Import Arcana auth
+// ...
+
+function createAuthService() {
+  // AUTH-1: Create a new instance of Arcana AuthProvider.
+  // ...
+
+  async function init() {
+    // AUTH-2: Initialize the wallet with App Mode (UI / NoUI).
+    // ...
```
Comment (karthik-durai) [inline src/services/auth.service.js:10]: add something like this - `const wallet ....`

File: index.html (hunk 1 above)
Comment (shrinathprabhu) [reply index.html]: Resolved at https://github.com/arcana-network/sdk-demo/pull/5/commits/58cfeb28f8e0ede2e9f9223a2410b107ef9f5a08

File: src/services/auth.service.js (hunk 2 above)
Comment (shrinathprabhu) [reply src/services/auth.service.js]: Resolved at https://github.com/arcana-network/sdk-demo/pull/5/commits/428e8ca5e474580e8e26199d9f9e502c2108b194

File: src/services/auth.service.js (hunk 3 above)
Comment (shrinathprabhu) [reply src/services/auth.service.js:10]: Resolved at https://github.com/arcana-network/sdk-demo/pull/5/commits/c6a0fe0dfbe2f54999bd47a2a35c6fa3d7b63447

**Pull request merged**